pyproject.toml

```diff
@@ -49,6 +49,13 @@
 [tool.black]
 line-length = 100
 
+[tool.isort]
```

Comment:
It's better not to put this in a shared config. Sections from multiple tools tend to snowball and are hard to locate when looking at the repo root.

Reply:
It is ok, especially for isort which will not grow. If a tool needs big/growing/verbose config keeping its config separated may be considered.

Reply:
It is not okay if you consider git merge driven updates from different upstreams.

Reply:
I understand both locations are functionally equivalent and from a little research both commonly used.  Rather than block this from moving forward due to the location, I'm going to leave it as is.  If in the future problems arise with it in pyproject.toml, it can be easily moved.